drivers/clock_control/clock_stm32_ll_h7.c

```diff
@@ -755,8 +755,11 @@ static int stm32_clock_control_init(const struct device *dev)
 	}
 #endif /* STM32_PLL3_ENABLE */
 
-	/* Set systick to 1ms */
-	SysTick_Config(CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC / 1000);
+	if (!IS_ENABLED(TICKLESS_KERNEL)) {
```

Comment:
I am actually uncertain which config parameter I should check here best. Maybe CORTEX_M_SYSTICK?

Reply:
I'm a bit puzzled here. Not sure why this problem is H7 only.
I need first to understand why this code is H7 only...

Reply:
Likely this should be completely removed. This is already done by `sys_clock_driver_init` in cortex_m_systick driver.
This would require some tests to confirm it though.

Reply:
@benediktibk I had some tests, and I think you can safely, fully, remove this line.

Reply:
I already played around a bit without this line on a H7, and also had no problems. It is removed.